athena-vertica/LICENSE.txt

```diff
@@ -0,0 +1 @@
+my license
```

Comment:
please update the license to use the same license as the other connectors. It just avoids confusion when someone comes across the older commits in your branch. Best practice is to include the real license as early as possible. I know this is an early CR though, so no worries.

Reply:
updated.